CHANGELOG.md

```diff
@@ -3,6 +3,8 @@
 All notable changes to this project will be documented in this file.
 
 ## [Unreleased]
+- [#103](https://github.com/os2display/display-client/pull/103)
+Removed "todo handle pagination" and handles pagination.
```

Comment:
Why not just "Update `PullStrategy` to handle pagination"? And maybe a new `@todo` that this should be fixed in the API?

Reply:
donnnneeee